infrastructure/scripts/check-license/Buildfile

```diff
@@ -4,3 +4,6 @@ metadata:
 spec:
   dependsOn:
   - ../../..
+  - ../../docker/builder
+  - ../../../services/cd-service
```

Comment:
Shouldn't that be redundant to `../../..`

Reply:
Found that there is an issue in execution plan in which it wasn't captured. so created story to test it out and fix it.

need to test after this merges if the depends on is working properly. For the time being fixed it with adding redundancy

Reply:
[SRX-7B29XV](https://revolution.dev/app/-JqFGExX46gs9mH7vxR5/-M37cZx7XYoOvQ5Mmcir/STORY/-N5_UFC3uri5Ovmf-wLv/)